app/uk/gov/hmrc/agentsubscriptionfrontend/views/invasive_check_start.scala.html

```diff
@@ -15,42 +15,42 @@
  *@
 
 @import uk.gov.hmrc.agentsubscriptionfrontend.config.AppConfig
-@import uk.gov.hmrc.agentsubscriptionfrontend.models.RadioWithInput
+@import uk.gov.hmrc.agentsubscriptionfrontend.models.RadioInvasiveStartSaAgentCode
 @import uk.gov.hmrc.agentsubscriptionfrontend.controllers.routes
 @import uk.gov.hmrc.play.views.html.helpers.{form, errorSummary}
 
-@(confirmResponseForm: Form[RadioWithInput])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(radioResponseForm: Form[RadioInvasiveStartSaAgentCode])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
-@uk.gov.hmrc.agentsubscriptionfrontend.views.html.main_template(appConfig, title =  error_prefix(confirmResponseForm) + Messages("invasive.title.saCode"), bodyClasses = None) {
+@uk.gov.hmrc.agentsubscriptionfrontend.views.html.main_template(appConfig, title =  error_prefix(radioResponseForm) + Messages("invasive.title.saCode"), bodyClasses = None) {
 
-    @if(confirmResponseForm.hasErrors) {
-        @errorSummary(Messages("error.summary.heading"), confirmResponseForm, Seq.empty, Some("confirmResponseSomethingTrackingGGA"))
+    @if(radioResponseForm.hasErrors) {
+        @errorSummary(Messages("error.summary.heading"), radioResponseForm, Seq.empty, Some("radioResponseSomethingTrackingGGA"))
```

Comment:
`confirmResponseSomethingTrackingGGA` changed to `"radioResponseSomethingTrackingGGA"`. 
It looks like that's a google analytics event name - did you mean to change this?